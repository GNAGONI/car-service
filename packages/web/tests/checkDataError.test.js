import React from 'react'
import renderer from 'react-test-renderer'
import CheckDataError from 'components/checkDataError'

it('Renders correctly', () => {
  const tree = renderer.create(<CheckDataError />).toJSON()
  expect(tree).toMatchSnapshot()
})
