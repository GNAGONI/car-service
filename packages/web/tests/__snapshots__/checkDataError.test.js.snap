// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="fade alert alert-danger show"
  role="alert"
>
  Ops something wrong!
</div>
`;
